daffodil-sapi/src/main/scala/org/apache/daffodil/sapi/Daffodil.scala

```diff
@@ -338,6 +339,13 @@ class ProcessorFactory private[sapi] (private var pf: SProcessorFactory)
     new DataProcessor(dp)
   }
 
+  class ExperimentalWrapper(val rootView: RootView)
+  /**
+   * Exposes the RootView object corresponding to this ProcessorFactory.  This can
+   * be used to start a walk using the walkFromRoot method in a DSOM Walker.
+   */
+  val experimental: ExperimentalWrapper = new ExperimentalWrapper(pf.rootView)
+
```

Comment:
I think it's reasonable to use the ``object experimental`` technique for the scala api. I think that makes fora cleaner scaladoc and feels more natural to scala users.